### DISCUSSION TITLE
I can't figure out how to get raw "error.data" from server request

### DISCUSSION THREAD

**Discussion by Trixpua** [nuxt/nuxt#25833] [Questions] [answered]
My application make call to an api and I am struggling to handle errors.

In server side I receive the correct "error.data" from api, but when I try to send the error to front end I receive a html code. 

How can I access the raw "error.data" without the html code to show the errors for the user?

https://stackblitz.com/edit/nuxt-starter-dwqm9z?file=app.vue

**Comment by TheAlexLichter** [comment, marked as answer]
~If you `throw` the error it should work fine.~

This is based on https://github.com/nuxt/nuxt/pull/25390. It was fixed on h3 but h3 was not released yet.
As workaround, set the correct header manually [StackBlitz](https://stackblitz.com/edit/nuxt-discussions-25833?file=server%2Fapi%2Fhello.ts,app.vue)

**Reply by Trixpua** [reply]
I found the same solution a few minutes ago, I was coming to write and saw your message, thank you very much!

**Reply by TheAlexLichter** [reply]
You are welcome 😋
